File: Python/Program/SivitasAkademik.py
from Human import Human


class SivitasAkademik(Human):

    __asal_universitas = ""
    __email_edu = ""

    def __init__(self, nik="", nama="", gender="", asal_universitas="", email_edu=""):
        super().__init__(nik, nama, gender)
        self.__asal_universitas = asal_universitas
        self.__email_edu = email_edu

    def set_asal_universitas(self, asal_universitas):
        self.__asal_universitas = asal_universitas

    def get_asal_universitas(self):
        return self.__asal_universitas

    def set_email_edu(self, email_edu):
        self.__email_edu = email_edu

    def get_email_edu(self):
        return self.__email_edu

    def display_data(self):
        super().display_data()
        print("| Asal Universitas    : ", self.__asal_universitas)
        print("| Email Edu           : ", self.__email_edu)

File: CPP/Program/SivitasAkademik.cpp
using namespace std;
#include <bits/stdc++.h>
#include "Human.cpp"
class SivitasAkademik : public Human
{
private:
    string asal_universitas;
    string email_edu;

public:
    // Constructor
    SivitasAkademik()
    {
        this->asal_universitas = "";
        this->email_edu = "";
    }

    SivitasAkademik(string nik, string nama, string gender, string asal_universitas, string email_edu) : Human(nik, nama, gender)
    {
        this->asal_universitas = asal_universitas;
        this->email_edu = email_edu;
    }

    // Setter & Getter

    void setAsalUniversitas(string asal_universitas)
    {
        this->asal_universitas = asal_universitas;
    }

    string getAsalUniversitas()
    {
        return asal_universitas;
    }

    void setEmailEdu(string email_edu)
    {
        this->email_edu = email_edu;
    }

    string getEmailEdu()
    {
        return email_edu;
    }

    void displayDaya()
    {
        Human::displayDaya();
        cout << "| Asal Universitas : " << this->getAsalUniversitas() << endl;
        cout << "| Email Edu        : " << this->getEmailEdu() << endl;
    }

    // Destructor
    ~SivitasAkademik() {}
};